src/framework/core/js/ModelTransformationTransforms.js

```diff
@@ -409,15 +433,19 @@ var fluid = fluid || fluid_2_0_0;
         }
 
         var outputArr = [];
-        fluid.each(options, function (arrVal, inPath) {
-            var val = fluid.model.transform.getValue(inPath, undefined, transformer);
+        fluid.each(input, function (val, key) {
             if (val === transformSpec.presentValue) {
-                outputArr.push(arrVal);
+                outputArr.push(options[key]);
```

Comment:
As you can see in line 416, I changed this to a standardInputTransformFunction, so the `input`/`inputPath` is automatically parsed by the framework - so that's the reason for removing the 'getValue' portion. The change in the each function (looping through inputs instead of options) was simply because it seemed more logical to look each input key up in the options block than vice versa.

I guess looking at it now, this result in a difference in behavior... Whereas before an input that was not present in the options block would just result in something not being present in the output array, this version will push an empty entry (ie. undefined) onto the array. We didn't have before, and dont have now any tests for this, and I'm not actually sure what the desired behavior should be